stdlib/os/__init__.pyi

```diff
@@ -117,7 +117,9 @@ if sys.platform != "win32":
     RTLD_LOCAL: int
     RTLD_NODELETE: int
     RTLD_NOLOAD: int
-    RTLD_DEEPBIND: int
+
+    if sys.platform != "darwin":
+        RTLD_DEEPBIND: int
```

Comment:
This is linux-specific (actually gnu libc specific, but typeshed doesn't really support GNU Hurd yet, lol). `dlopen(3)` linux man page says: "The RTLD_NOLOAD, RTLD_NODELETE, and RTLD_DEEPBIND flags are GNU  extensions; the first two of these flags are also present on Solaris."